index.bs

```diff
@@ -1401,14 +1447,18 @@ input parameters:
 - A list of {{PublicKeyCredential}} objects provided by the [RP] with the intention that, if any of these are known to the
     authenticator, it should not create a new credential.
 - Extension data created by the client based on the extensions requested by the [RP].
+- The |requireResidentKey| parameter of the |options|.{{MakeCredentialOptions/authenticatorSelection}} dictionary.  
 
 When this operation is invoked, the authenticator must perform the following procedure:
 - Check if all the supplied parameters are syntactically well-formed and of the correct length. If not, return an error code
-    equivalent to UnknownError and terminate the operation.
+    equivalent to UnknownError and terminate the operation.    
 -  Check if at least one of the specified combinations of {{PublicKeyCredentialType}} and cryptographic parameters is supported.
     If not, return an error code equivalent to NotSupportedError and terminate the operation.
 - Check if a credential matching any of the supplied {{PublicKeyCredential}} identifiers is present on this authenticator. If
     so, return an error code equivalent to NotAllowedError and terminate the operation.
+- If the |requireResidentKey| flag is set to |true| and the authenticator can store [=Client-side-resident Credential 
+    Private Key=], continue to the next step. Otherwise, return an error code equivalent to QuotaExceededError and  
```

Comment:
This repeats the mistake in the *requireResidentKey* step above: a false value will cause a `QuotaExceededError`.